### PR TITLE
Add some ommissions to CHANGELOG and update some swarm commands

### PR DESCRIPTION
**Pull request by tiborvass**

### CODE REVIEW

**Comment by thaJeztah** [comment]
LGTM

**Comment by mavenugo** [comment]
LGTM

**Comment by dmcgowan** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -107,9 +111,9 @@ be found.
 
 ### Swarm Mode
 
-+ New `swarm` command to manage swarms with `init`, `join`, `leave`, `update` subcommands [#23361](https://github.com/docker/docker/pull/23361)
-+ New `service` command to manage swarm-wide services with `create`, `inspect`, `update`, `remove`, `tasks` subcommands [#23361](https://github.com/docker/docker/pull/23361)
-+ New `node` command to manage nodes with `accept`, `promote`, `demote`, `inspect`, `update`, `tasks`, `ls` and `rm` subcommands [#23361](https://github.com/docker/docker/pull/23361)
++ New `swarm` command to manage swarms with `init`, `join`, `join-token`, `leave`, `update` subcommands [#23361](https://github.com/docker/docker/pull/23361)
```
Comment (vieux) [inline CHANGELOG.md]: `join-token` was added in  https://github.com/docker/docker/pull/24823

File: CHANGELOG.md [hunk 2]
```diff
@@ -107,9 +111,9 @@ be found.
...
++ New `service` command to manage swarm-wide services with `create`, `inspect`, `update`, `rm`, `ps` subcommands [#23361](https://github.com/docker/docker/pull/23361)
```
Comment (vieux) [inline CHANGELOG.md]: `ps` was added in https://github.com/docker/docker/pull/25140

File: CHANGELOG.md [hunk 3]
```diff
@@ -107,9 +111,9 @@ be found.
...
++ New `node` command to manage nodes with `accept`, `promote`, `demote`, `inspect`, `update`, `ps`, `ls` and `rm` subcommands [#23361](https://github.com/docker/docker/pull/23361)
```
Comment (vieux) [inline CHANGELOG.md]: `ps` was added in https://github.com/docker/docker/pull/25140

**Comment by tiborvass** [comment]
@vieux fixed

**Comment by vieux** [comment]
LGTM

**Comment by tiborvass** [comment]
Ref #25238

**Pull request merged**